.github/workflows/check.yaml

```diff
@@ -12,13 +12,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - uses: subosito/flutter-action@v1
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
```

Comment:
Please prefer _convention over configuration_. This is [the default](https://github.com/subosito/flutter-action/blob/5e1529bc12adae5b26f30f0a4046ac743d6f0c5d/action.yml#L14) value for `channel` so does not need to be specified. Same goes for other instances in this PR.

Reply:
Personally, I prefer to specify values like that to make it clear that the step should always run with latest stable version, but if the convention is different, that's fine 🙂 Removed those definitions in https://github.com/ably/ably-flutter/pull/326/commits/34fbc661f27552ba14413eebae362848f110bc7c